tests/Integration/Classes/controller/AdminControllerTest.php

```diff
@@ -87,6 +87,27 @@ protected function tearDown(): void
         ServiceLocator::setServiceContainerInstance($this->savedContainer);
     }
 
+    /**
+     * Check if html in trans is not escaped when the _raw parameter is used
+     *
+     * @dataProvider getControllersClasses
+     *
+     * @param string $controllerClass
+     *
+     * @return void
+     */
+    public function testTrans(string $controllerClass): void
+    {
+        $testedController = new $controllerClass();
+        $transMethod = new \ReflectionMethod($testedController, 'trans');
```

Comment:
Hmm since you test it here the same way as in ObjectModel, my initial comment about a bridge method makes less sense At least if you use this everywhere there's some consistency in the code